src/status_im/anon_metrics/transformers.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.anon-metrics.transformers)
```

Comment:
could you please add comments , why we need this, thanks

Reply:
@flexsurfer done